src/content/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const LandingPage = () => <div>LANDING PAGE</div>;
```

Comment:
Thie LandingPage const should be replaced with `const LandingPage = () => {
  return <div>LANDING PAGE</div>;
}`

See https://www.carbondesignsystem.com/tutorial/react-step-1#import-and-export-content-pages

Reply:
It's the same, but ok